include/bmp.h

```diff
@@ -0,0 +1 @@
+void hello(void);
```

Comment:
There is no need to use write void in between the parenthesis when the function takes no parameters. That's a prehistoric C convention. You should just do:
`void hello();`

Reply:
yep should've known that. thanks